services/processor/native/call.go

```diff
@@ -136,7 +136,14 @@ func (s *service) prepareMethodInputArgsForCall(methodInstance types.MethodInsta
 	}
 
 	if i < methodType.NumIn() {
+		if methodType.NumIn()-1 == i { // missing a single argument
+			if k := methodType.In(methodType.NumIn() - 1).Kind(); k == reflect.Slice {
+				return res, nil
+			}
+		}
+
 		return nil, errors.Errorf("method '%s' takes %d args but received less", functionNameForErrors, methodType.NumIn())
+
```

Comment:
in rows 110, 112, 114, 116 there is no check that `arg` is in agreement with `Elem().Kind()`. consider adding tests for cases where an array or variadic are expected, but a different type is passed in

Reply:
Fixed.